### ISSUE TITLE
load error

### ISSUE THREAD

**Issue by FerretDave** [BahamutDragon/pcgen#964]
using PCgen v6.08.00 RC8 and your 'master' dataset, I get the following errors on start:
Should I be using a different version of PCGen ?

21:26:44.213 INFO main Main:138 Starting PCGen v6.08.00 RC8
21:26:44.319 INFO main LanguageBundle:134 Initialising language bundle with locale en_GB.
21:26:48.326 LSTERROR main LstUtils:147 Illegal Token 'ALIGNMENTFEATURE' 'YES' for pcgen.cdom.inst.CodeControl Controller in file:/C:/Utils/PCGe/PCGen/6.08.00RC8/system/gameModes/Bahamut35e/codeControl.lst
21:26:48.327 LSTERROR main LstUtils:147 Illegal Token 'DOMAINFEATURE' 'YES' for pcgen.cdom.inst.CodeControl Controller in file:/C:/Utils/PCGe/PCGen/6.08.00RC8/system/gameModes/Bahamut35e/codeControl.lst
21:27:06.680 INFO Thread-2 SourceFileLoader:534 Loading game Bahamut35e and sources [Campaign: Sun Has Gone].
21:27:06.759 LSTERROR Thread-2 CDOMControlLoader:146 DEFAULTVARIABLEVALUE found an unsupported format: HANDED (Source: file:/C:/Utils/GIT/pcgen/data/35e/wizards_of_the_coast/core/universal_rules/ur__datacontrols.lst )
21:27:06.759 LSTERROR Thread-2 CDOMControlLoader:146 Failed in parsing typeStr: DEFAULTVARIABLEVALUE HANDED|Right for pcgen.cdom.content.DefaultVarValue HANDED Right
21:27:06.760 LSTERROR Thread-2 CDOMControlLoader:146 Illegal Token 'DEFAULTVARIABLEVALUE' 'HANDED|Right' for pcgen.cdom.content.DefaultVarValue HANDED Right in file:/C:/Utils/GIT/pcgen/data/35e/wizards_of_the_coast/core/universal_rules/ur__datacontrols.lst

This is using my own campaign file which has multiple sources, however loading *just* the Wizards of the coast core books gives me most of the above - specifically:
21:31:04.567 INFO Thread-4 SourceFileLoader:534 Loading game Bahamut35e and sources [Dungeons & Dragons - Core Books].
21:31:04.577 LSTERROR Thread-4 CDOMControlLoader:146 DEFAULTVARIABLEVALUE found an unsupported format: HANDED (Source: file:/C:/Utils/GIT/pcgen/data/35e/wizards_of_the_coast/core/universal_rules/ur__datacontrols.lst )
21:31:04.578 LSTERROR Thread-4 CDOMControlLoader:146 Failed in parsing typeStr: DEFAULTVARIABLEVALUE HANDED|Right for pcgen.cdom.content.DefaultVarValue HANDED Right
21:31:04.578 LSTERROR Thread-4 CDOMControlLoader:146 Illegal Token 'DEFAULTVARIABLEVALUE' 'HANDED|Right' for pcgen.cdom.content.DefaultVarValue HANDED Right in file:/C:/Utils/GIT/pcgen/data/35e/wizards_of_the_coast/core/universal_rules/ur__datacontrols.lst

(I do have the system/gamemode files copied over for gamemode Bahamut35e)

**Comment by FerretDave** [comment]
additional - loading just the core dnd books using the Bahamut35e gamemode, I get a java NPE if I try and create a new character.

Have tried with 6.07.08, 6.08.00RC7, 6.08.00RC8, 6.09.01 *and* 6.09.03 (both the 6.09 versions seem weird generally, not sure they like java 11 really - things like the log window wont resize)

Switching back to 35e gamemode lets me continue using pcgen and my normal sources for now.
Using java 11, is it worth trying openjdk13 ?

**Comment by BahamutDragon** [comment]
Master Branch needs the latest version of 6.09. 6.08 Branch works only with 6.08 or earlier versions.

Mixing the branches will cause all sorts of issues. 

6.09 is moving the UI to JavaFX and that has had issues in general. For the PCGen specific issues with resizing, you need to talk to the pcgen volunteers if they are still active.

**Comment by BahamutDragon** [comment]
Any update on this @FerretDave ?

**Comment by FerretDave** [comment]
ok, while I've got 6.09.03 able to actually load and create characters with gamemode Bahamut35e now,  it still logs SEVERE JavaFX exceptions so I am wary.
Will probably roll back to 6.08.00RC8 and checkout the right branch of your code.
So - all the PR's I've logged previously were with master (as was generally using the autobuild) - do I need to resubmit those into 6.08, or can they be brought in easily? 
And if I submit future PR's into 6.08, how would I go about testing them into master for 6.09?
